iosxr_iso2vbox.py

```diff
@@ -401,6 +441,11 @@ def xr_cli_wait_for_output(command, pattern):
         logger.debug('Waiting 30 seconds...')
         time.sleep(30)
 
+        logger.info("Issuing shutdown command")
+        child.sendline("run shutdown -P now")
+        logger.debug('Waiting 10 seconds...')
+        time.sleep(10)
```

Comment:
Prompt reappears immediately as shutdown is kicked off asynchronously. I could wait for the prompt but it doesn't add any value.

```
[xr-vm_node0_RP0_CPU0:~]$shutdown -P now
[xr-vm_node0_RP0_CPU0:~]$[20:22:39.616] Sending KILL signal to ds..
[20:22:39.624] Sending KILL signal to processmgr..
PM disconnect successStopping OpenBSD Secure Shell server: sshdinitctl: Unknown instance:
Stopping system message bus: dbus.
hwclock: can't open '/dev/misc/rtc': No such file or directory
Stopping system log daemon...0
Stopping internet superserver: xinetd.
Stopping crond: OK
Stopping rpcbind daemon...
done.
Stopping Lighttpd Web Server: stopped /usr/sbin/lighttpd (pid 2083)
lighttpd.
Libvirt not initialized for container instance
Deconfiguring network interfaces... done.
Sending all processes the KILL signal...
Deactivating swap...
Unmounting local filesystems...
Connection closed by foreign host
```

I *could* check for the 'unmounting local filesystems' line, but I'm not sure if I may not even see it depending on the timing of things.